MeasureAuthoringTool/mat-drupal
# HARP ID countdown banner

## Description
Create a Site Status block type to be able to place a site-wide message, including an optional countdown timer.

## To Test
Import config, clear cache.
Go to block layout, and add a new Site Status block, probably to the header region. Configure as desired and save, see that it looks something like the attached screenshot.

<img width="1075" alt="Screen Shot 2020-04-14 at 10 27 58 AM" src="https://user-images.githubusercontent.com/1919263/79243017-af271b00-7e3a-11ea-83cb-0e956a8c13ac.png">
